### DISCUSSION TITLE
Link title editing

### DISCUSSION THREAD

**Discussion by ryanherford** [outline/rich-markdown-editor#245] [General]
Editing link titles in the editor isn't a straightforward process. Changing the location is simple, but for the title we have to do some text shimmying to keep the link active.

**Comment by tommoor** [comment]
Any chance you could provide a video / gif of the troubles? Might help to figure out how it can be improved!

**Reply by ryanherford** [reply]
No problem 😄 

It could be a problem with my specific configuration. But my current use-case involves editing and updating link titles a lot. Potentially the ability to edit the raw markdown, or an additional field for the title might help?

Thanks!


![ezgif-6-f28f06a6e61e](https://user-images.githubusercontent.com/18262949/88768173-0a003a80-d1be-11ea-8288-a8eaac5ca263.gif)
